Update SRM_logging_config.md (#1956)
SkylineCommunications/dataminer-docs: user-guide/Standard_Apps/SRM/srm_advanced_config/SRM_logging_config.md

<!-- RN 31988 -->

> From within an LSO script, you can add custom records to the action logs or debug logs, together with a severity indication for action logs. All custom records added to the debug logs will be of severity Info.

Reviewer: OK to rephrase like this?
```suggestion
From within an LSO script, you can add custom records to the action logs or debug logs. For action logs, these come with a custom severity indication. For the debug logs, custom records are always of severity Info.
```

Author: ok
